packages/flutter_tools/bin/podhelper.rb

```diff
@@ -73,9 +73,13 @@ def flutter_additional_ios_build_settings(target)
 
 # Same as flutter_ios_podfile_setup for macOS.
 def flutter_additional_macos_build_settings(target)
-  print target.platform_name
   return unless target.platform_name == :osx
 
+  # [target.deployment_target] is a [String] formatted as "10.8". Check if this is < 10.9
+  # See MACOSX_DEPLOYMENT_TARGET note below.
+  deployment_target_major, deployment_target_minor = target.deployment_target.match(/(\d*).?(\d*)/).captures
```

Comment:
should these `*`'s be `+`'s?

Reply:
should we validate that there are in fact two matches?

Reply:
> should these `*`'s be `+`'s?

No, then "11" is captured as "1" and "1".

<img width="184" alt="Screen Shot 2020-12-22 at 10 40 42 AM" src="https://user-images.githubusercontent.com/682784/102922007-3012c980-4442-11eb-972c-62b77cbbe1dd.png">
<img width="158" alt="Screen Shot 2020-12-22 at 10 40 52 AM" src="https://user-images.githubusercontent.com/682784/102922010-3143f680-4442-11eb-842a-791035e8b391.png">


> should we validate that there are in fact two capture group matches?

If there's only one capture  `deployment_target_minor` will be set to "", which is coerced to 0.  So "11" is "11" and "0" which is what we want.

Reply:
`/(\d+).?(\d*)/` might be more precise though; there does need to be a first number.